### PR TITLE
Use cairo as a dependency of Graphviz

### PR DESCRIPTION
**Pull request by jakirkham**

### CODE REVIEW

**Comment by ccordoba12** [comment]
Simple question: Why cairo is needed by Graphviz?

**Comment by jakirkham** [comment]
Here is a list of Graphviz's dependencies ( http://www.graphviz.org/Download_source.php ). However, you may note that nearly all (looks like all) are optional. This includes ones we have added like `freetype`. So, need is not really the right word here. However, Graphviz's strength is in how it is able to export into different formats for docs, debugging, dependency tracing, etc. As is common with these graphs, they can get quite big and unwieldy so being able to zoom and move around is important.

TL;DR `cairo` is very helpful as it provides lossless formats that are good for interactive viewing of large detailed diagrams, namely, SVG, PDF, PS.

**Comment by ccordoba12** [comment]
Thanks for the detailed explanation! Merging then!

**Comment by jakirkham** [comment]
Hope it wasn't too much. Thanks @ccordoba12.

**Comment by ccordoba12** [comment]
It was perfect!! Don't worry about it ;-)
